packages/react-events/src/dom/Keyboard.js

```diff
@@ -16,26 +16,32 @@ import React from 'react';
 import {DiscreteEvent} from 'shared/ReactTypes';
 import type {ReactEventResponderListener} from 'shared/ReactTypes';
 
-type KeyboardEventType = 'keydown' | 'keyup';
+type KeyboardEventType = 'keyboard:keydown' | 'keyboard:keyup';
```

Comment:
I guess the keyboard prefix is because we're also adding the keyboard pointer type. What do you expect this to expand to in the future?

Reply:
I think we should scope event names to their responder, especially given that `keydown` and `keyup` are also the names of native events